Elena_Poleschuk/Module 3 -  Java/Week 1/Hometask3/src/by/epam/tat/lecture1/list/List.java

```diff
@@ -0,0 +1,56 @@
+package by.epam.tat.lecture1.list;
+
+import java.util.*;
+
+/**	
+ * Sort list or array by string length 
+ *
+ * @author Elena
+ *
+ */
+public class List {
+	
+	public void createList(){
+		Scanner in = new Scanner (System.in);
+		System.out.println("How many strings do you want to create?");
+		int nString = in.nextInt();
+		in.nextLine();  // Consume newline left-over
+		String[] masStrings = new String[nString]; 
+		for (int n = 0; n < nString; n++){
+			System.out.println("Enter " + (n + 1) + " srting");
+			masStrings[n] = in.nextLine();	
+		}	
+		System.out.println("The List is created");
+		System.out.println("-----------------");
+		in.close();
+		printList(masStrings);
+		sorting(masStrings);
```

Comment:
It's not right to call the methods as chain.
I recommend to write methods in main method in series